yarn.lock

```diff
@@ -15944,7 +15953,7 @@ resolve@^2.0.0-next.3:
   version: 3.11.1
   resolution: "webpack-dev-server@npm:3.11.1"
   dependencies:
-    ansi-html: 0.0.7
+    ansi-html: 0.0.8
```

Comment:
@kpierre13 how did you get it to make this update? When I deleted the whole "webpack-dev-server@npm:3.11.1": block, it did not update in ansi-html after I did `yarn install` (following the directions in https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance). I don't think we can just edit dependency versions ourselves, as the libraries define what versions they define on (and theoretically there could be an incompatibility with 0.0.8).

Reply:
In fact, when I go back to what is on dev and either follow the documented process or run `yarn up -R ansi-html` as suggested by @aednichols, neither of these usages update. So I don't think the upgrade can be done without upgrading the libraries that depend on ansi-html.

See the wiki document for options in this case.

Reply:
I also ran that command and didn't see any changes, so I assumed that I must have to change it manually. Are the circleci checks not in place for version changes? How would I go about testing to make sure that these version upgrades don't break anything?

Reply:
Notably, both instances of this are only used for serving a local dev environment, so we could close this warning as not used. Upgrading would require going to the new major version of `react-scripts`, which is a nontrivial undertaking.

Reply:
Correct, and this is documented in the wiki page (**A lot of the Dependabot alerts can be safely dismissed. These are all valid reasons for dismissing the alerts...**).

Reply:
I understand. So if everyone agrees, I'm going to close this PR with the resolution to simply dismiss this alert due to its non-usage in production.